Python/ceval_macros.h

```diff
@@ -151,17 +151,15 @@ GETITEM(PyObject *v, Py_ssize_t i) {
     } while (0)
 #define JUMPTO(x)       do { \
                             next_instr = _PyCode_CODE(_PyFrame_GetCode(frame)) + (x); \
-                            frame->next_instr_offset = 0; \
+                            assert(frame->return_offset == 0); \
```

Comment:
Is this assert safe? What if we somehow got here after making and returning from a `CALL`, which would have set it to some nonzero value? Or is it reset to zero after consuming? (Hmm... It *seems* to be. Can we trust that? Does it ever actually *need* to be reset except to make this assert pass?)